CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog of FOSSology
 
+### 4.1.0 (May 12th 2022)
+
+This release adds important corrections to
+[4.0.0](https://github.com/fossology/fossology/releases/tag/4.0.0)
```

Comment:
This release adds important corrections to [4.1.0-rc1](https://github.com/fossology/fossology/releases/tag/4.1.0-rc1)

Reply:
Done, please check.